### ISSUE TITLE
Maya 2017 linux 

### ISSUE THREAD

**Issue by Dimko1990** [KhronosGroup/OpenCOLLADA#601]
cant install. no COLLADAMaya.so in all directorys

**Comment by Dimko1990** [comment]
version from PC have error on load
invalid ELF header

**Comment by RemiArnaud** [comment]
indeed, there is no build for Maya for Linux. I don't know if this builds, you can try from sources
